Contributions-4-2-Ind.md

```diff
@@ -1,62 +1,11 @@
 ---
 layout: default
-title: Engaging in a FOSS project as a company employee
+title: Open Source Contributions
```

Comment:
The Contributions-4-1-Ind.md and Contributions-4-2-Ind.md , seems to have the same title "Open Source Contributions".  I am not good for Markdown chapters. What is the intention at the file level? Could we keep here the subchapter title only?

Reply:
Good question. This entire front matter (between the dashes) is only used by the markdown rendering engine and the resulting theme. In our case, the title attribute is not used (and rendered) at all by the selected theme.

Take a look at the resulting GitHub page: https://expertlearninglab.github.io/foss-learning/Contributions-4-Ind (change the URL to check the other sections).

So, we could either remove the title or just keep a consistent naming. I renamed the title here to make it consistent with the other sections / files.

Reply:
Let us keep it for a consistent naming